scripts/lib/herold.py

```diff
@@ -0,0 +1,24 @@
+import subprocess
+import tempfile 
+
+def convert(html: str) -> str:
+    tmp_file = tempfile.NamedTemporaryFile(
+        mode="w+t",
+        encoding="UTF-8",
+    )
+    
+    result = subprocess.run(
+        ["herold", "-o", tmp_file.name],
+        capture_output=True,
+        input=html,
+        text=True,
+        encoding="UTF-8",
+    )
+
+    if result.returncode != 0 or 'herold Version' in result.stdout:
+        raise RuntimeError("Convertation from html to xml failed: ", result.stderr)
+    
+    xml = tmp_file.read()    
+    tmp_file.close()
```

Comment:
Nit: for not do it manually [context manager can be used](https://docs.python.org/3/library/tempfile.html#examples):
```
with tempfile.TemporaryFile() as fp:
    fp.write(b'Hello world!')
```